src/databricks/labs/lsql/dashboard.py

```diff
@@ -1,27 +1,31 @@
 import json
 from pathlib import Path
-from typing import Protocol, ClassVar, runtime_checkable
+from typing import ClassVar, Protocol, runtime_checkable
 
 import sqlglot
 import yaml
 from databricks.sdk import WorkspaceClient
 from databricks.sdk.service.workspace import ExportFormat
 
-from databricks.labs.lsql.lakeview import Dashboard, Page, Query, NamedQuery, ControlFieldEncoding
+from databricks.labs.lsql.lakeview import ControlFieldEncoding
+from databricks.labs.lsql.lakeview import Dashboard as LakeviewDashboard
```

Comment:
that's exactly why I named the class as `Dashboards` - to prevent name collisions.